Makefile

```diff
@@ -19,8 +20,8 @@ test: all
 %.json: %.YAML-tmLanguage
 	$(CONVERT) $< > $@
 
-$(CONVERT):
-	npm install js-yaml
-
```

Comment:
As far as I could tell, this rule wasn't doing anything useful. It was actually creating two different targets (`npx` and `js-yaml`) but not using them as prereqs for anything. As long as `npm` is installed on the developer machine, `npx` should be sufficient to fetch any of the tools necessary, so I don't think we need a make rule to add them.